app/src/main/java/com/oracle/javafx/scenebuilder/app/i18n/I18N.java

```diff
@@ -33,15 +33,34 @@
 
 import java.text.MessageFormat;
 import java.util.ResourceBundle;
-
-/**
- *
- */
-
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.Locale;
+import java.util.PropertyResourceBundle;
 
 public class I18N {
     
     private static ResourceBundle bundle;
+
+    private static ResourceBundle.Control UTF8_ENCODING_CONTROL = new ResourceBundle.Control() {
```

Comment:
The app and kit modules has different properties file and this problem has high urgency. (Because Japanese people can't use SceneBuilder now... due to Mojibake.) I think that we need to consider about how to integrate two I18N class. 
So I would like to file another issue about integration between two I18N class and work that issue.